src/Accordion.php

```diff
@@ -170,9 +185,12 @@ public function renderItem($header, $item, $index)
             $options = ArrayHelper::getValue($item, 'contentOptions', []);
             $options['id'] = $id;
             Html::addCssClass($options, ['widget' => 'collapse']);
-            if ($index === 0) {
-                Html::addCssClass($options, 'show');
+
+            // check if accordion expanded, if true add show class
+            if($this->_isAccordionExpanded($item)){
```

Comment:
There should be spaces after `if` statement and between `)` and `{` according [Yii code styles](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#54-control-statements).